src/control/server/storage/bdev/provider.go

```diff
@@ -207,12 +108,12 @@ func forwardScan(req ScanRequest, cache *ScanResponse, scan scanFwdFn) (msg stri
 	return
 }
 
-// Scan attempts to perform a scan to discover NVMe components in the
+// storage.BdevScan attempts to perform a scan to discover NVMe components in the
 // system. Results will be cached at the provider and returned if
 // "NoCache" is set to "false" in the request. Returned results will be
 // filtered by request "DeviceList" and empty filter implies allowing all.
-func (p *Provider) Scan(req ScanRequest) (resp *ScanResponse, err error) {
-	if p.shouldForward(req) {
+func (p *Provider) Scan(req storage.BdevScanRequest) (resp *storage.BdevScanResponse, err error) {
+	/*if p.shouldForward(req) {
```

Comment:
Should probably just go ahead and remove these commented-out blocks. I think I left them in for reference/scaffolding while this was in progress.

Reply:
Alright.